pymongoexplain/commands.py

```diff
@@ -24,28 +24,29 @@
 Document = Union[dict, SON]
 
 class BaseCommand():
-    def __init__(self, dictionary):
-        if type(self) == BaseCommand:
-            raise NotImplementedError
-        self.command_document = self.convert_to_camelcase(dictionary)
-
-    def convert_to_camelcase(self, d: dict) -> dict:
+    def __init__(self, command_name, collection):
+        self.command_document = {}
+        self.command_name = command_name
+        self.collection = collection
+
+    def convert_to_camelcase(self, d):
+        if not isinstance(d, dict):
+            return d
         ret = dict()
         for key in d.keys():
             if d[key] is None:
                 continue
             new_key = key
-            if "_" in key:
+            if "_" in key and key != "_id":
                 new_key = key.split("_")[0] + ''.join(
                     [i.capitalize() for i in key.split("_")[1:]])
             if type(d[key]) == list:
```

Comment:
Can you change this to `isinstance(..., list)`? (and change all the other usages of `type` to use isinstance too?)

Reply:
Done.